src/pvrclient-mediaportal.cpp

```diff
@@ -661,30 +701,18 @@ PVR_ERROR cPVRClientMediaPortal::GetChannels(ADDON_HANDLE handle, bool bRadio)
   if( !SendCommand2(command, lines) )
     return PVR_ERROR_SERVER_ERROR;
 
-#ifdef TARGET_WINDOWS
+#ifdef TARGET_WINDOWS_DESKTOP
```

Comment:
this broke uwp

Reply:
Whoops. Should be fixed with #91. Thanks for letting me know...